app.js

```diff
@@ -1,97 +1,140 @@
+  }
+  document.getElementById('load-more-button').style.display = 'block';
+  document.getElementById('hide-button').style.display = 'none';
 }
 
-const members_url =
-  "https://api.github.com/orgs/move-fast-and-break-things/members";
+const members_url = "https://api.github.com/orgs/move-fast-and-break-things/members";
 
+// fetch authors data
 async function getAuthors(apiURL) {
-  let authors = [];
-  const result = await fetch(apiURL); // Using github api to get the list of members
-  const data = await result.json();
-  for (const element of data) {
-    // Getting avatar name, biography and url from each user, then putting them into an array
-    const result = await fetch(element.url);
-    const data = await result.json();
-    authors.push({
-      name: data.name ? data.name : data.login,
-      bio: data.bio,
-      avatar: data.avatar_url,
-      html_url: data.html_url,
-    });
+  try {
+    const response = await fetch(apiURL);
+    if (!response.ok) {
+      throw new Error(`HTTP error! status: ${response.status}`);
+    }
+    const data = await response.json();
+    return Promise.all(data.map(async (element) => {
+      const userResponse = await fetch(element.url);
+      if (!userResponse.ok) {
+        throw new Error(`HTTP error! status: ${userResponse.status}`);
+      }
+      const userData = await userResponse.json();
+      return {
+        name: userData.name || userData.login,
+        bio: userData.bio,
+        avatar: userData.avatar_url,
+        html_url: userData.html_url,
+      };
+    }));
+  } catch (error) {
+    console.error("Error fetching authors:", error);
+    return [];
   }
-  return authors;
 }
 
+// set authors in the DOM
 async function setAuthors(apiURL) {
-  // Getting authors and placing them
-  const authors = await getAuthors(apiURL);
-  for (const author of authors) {
-    document.querySelector("#creators > div > div").innerHTML += `
-    <div class="person">
-      <div class="name_and_img">
-        <img src="${author.avatar}" alt="">
-        <p class="person_name">${author.name}</p>
-      </div>
-      <p class="about_person">${author.bio}</p>
-     </div>
-     `;
+  try {
+    const authors = await getAuthors(apiURL);
+    const container = document.querySelector("#creators > div > div");
+    if (!container) {
+      throw new Error("Container element not found");
+    }
+    authors.forEach(author => {
+      const personDiv = document.createElement('div');
+      personDiv.className = 'person';
+      personDiv.innerHTML = `
+                <div class="name_and_img">
+                    <img src="${author.avatar}" alt="${author.name}'s avatar">
+                    <p class="person_name">${author.name}</p>
+                </div>
+                <p class="about_person">${author.bio || 'No biography available'}</p>
+            `;
+      container.appendChild(personDiv);
+    });
+  } catch (error) {
+    console.error("Error setting authors:", error);
```

Comment:
for this function, because we aren't doing anything besides the error handling here, let's remove the `try...catch` to and add error handling when we call it?